redisinsight/api/src/modules/workbench/workbench.service.ts

```diff
@@ -50,6 +51,24 @@ export class WorkbenchService {
     return this.commandExecutionProvider.create(commandExecution);
   }
 
+    /**
+   * Send redis command from workbench and save history
+   *
+   * @param clientOptions
+   * @param dto
+   */
+     async createBunchCommandsExecution(
```

Comment:
You are creating multiple command executions but not Bunch*ComandExecution :) 
Please rename to createCommandExecutions